server_flask/database.py

```diff
@@ -0,0 +1,83 @@
+import requests
+import requests.auth
+import requests.exceptions
+from urllib.parse import urljoin
+
+
+class CouchDBClientConfig:
+    """
+    Configuration for connection to a CouchDB instance.
+    """
+
+    _baseurl: str
+    _admin_password: str
+    _admin_user: str
+
+    def __init__(self, *, baseurl: str, admin_user: str, admin_password: str):
+        self._baseurl = baseurl
+        self._admin_user = admin_user
+        self._admin_password = admin_password
+
+    @staticmethod
+    def load(baseurl: str, admin_user: str, admin_password: str):
+
+        return CouchDBClientConfig(
+            baseurl=baseurl,
+            admin_user=admin_user,
+            admin_password=admin_password,
+        )
+
+    @staticmethod
+    def initialize(self):
```

Comment:
The database initialize only ever happens one time, when I set up the cluster / database.  The invoke task seems the right place for that, don't think we need it here in Flask at all.

Reply:
Removed this and pushed the changes.